### ISSUE TITLE
(bug) Selectors with special characters in them are not working properly

### ISSUE THREAD

**Issue by shobhitchittora** [checkly/headless-recorder#67]
My page has a selector named `.scTrack:Aheader_home` which is changed to the below querySelector by the extension.

```js
await page.type('.css-htcxmu > .css-9tec82 > .css-daw4m5 > .css-1o4f6ww > .scTrack\3Aheader_home', 'undefined')
```

As you can see `:` is changed to `\3A`, which doesn't work while the query selector runs.

**Comment by jiegec** [comment]
I encountered the same issue:

```js
    await page.waitForSelector('.monaco-scrollable-element .codelens-decoration:nth-child(3) > #\30')
    await page.click('.monaco-scrollable-element .codelens-decoration:nth-child(3) > #\30')
```

It should be:

```js
    await page.waitForSelector('.monaco-scrollable-element .codelens-decoration:nth-child(3) > #\\30')
    await page.click('.monaco-scrollable-element .codelens-decoration:nth-child(3) > #\\30')
```

**Comment by ianaya89** [comment]
Fixed with #126
